docs/DeveloperGuide.md

```diff
@@ -661,62 +661,24 @@ testers are expected to do more *exploratory* testing.
-```
-
-Step 3. If the user intended to add a task with that deadline, then he/she would continue but if it was not intended, they would be alerted to the issue and be able to delete the task.
-
-### \[Proposed\] Multiple employees assigned to each task
-
-#### Proposed Implementation
-
-In this version of TaskHub, only 1 employee can be assigned at a time to each task.
-
-Tasks in a project can be worked on by multiple people at a time, so it would be more appropriate to expand the task to take on more than 1 employee.
-
-The current implementation of tasks is already using a `List` that has been restricted to hold only one `Employee`.
-
-`List` was chosen as the data type for storing assigned `Employees` in preparation for a future iteration where more employees could be held in each task.
-
-Thus, expanding `Task` to take more than one employee would simply involve allowing the list to take more than 1 `Employee`.
-
-However, this was not done in the current implementation due to the already complex nature of the `assignT` command which has to modify multiple instances of objects stored in the model.
-
-### \[Proposed\] Allowing multiple spaces between indexes
-
-#### Proposed Implementation
-
-TaskHub is not currently able to handle multiple spaces between indexes.
-
-In order to deal with multiple spaces, the following solution can be used (taken from [this StackOverflow discussion](https://stackoverflow.com/questions/2932392/java-how-to-replace-2-or-more-spaces-with-single-space-in-string-and-delete-lead))
-
-```Java
-String after = before.trim().replaceAll(" +", " ");
-```
-
-This will allow indexes with multiple spaces between indexes to be handled automatically instead of the user having to find where they may have put the additional space(s).
-
+## **Appendix: Planned Enhancements**
+
+Given below are the planned enhancements for the application.
+
+1. The current validation for duplicate employees is too strict as it does not allow employees with the same name, but different details, to exist in the employee list.
+We plan to add a unique ID field for each employee to allow employees with the same name and differing IDs to exist in the employee list.
+2. The current validation for emails is too lenient as it allows some invalid emails to be accepted as valid.
+For example, in the following email format `local-part@domain`, the `local-part` can exceed the maximum length of 64 characters and is considered valid in our application, when it shouldn't be.
+We plan to use a stricter regular expression to validate emails, which is mentioned [here](https://www.baeldung.com/java-email-validation-regex).
+3. Currently, there is no validation to check if the project deadline is before the task deadline.
+Therefore, users can add tasks with deadlines past the project deadline by accident without knowing.
+We plan to add a warning to the user if the user tries to add a task with a deadline past the project deadline:
```

Comment:
Might it be better to have such code formatted as blocks rather than inline?

We plan to add a warning to the user if the user tries to add a task with a deadline past the project deadline:

```
The newly added task has a deadline past the project deadline! Check again if the details are correct and edit if needed!
```